presto-main/src/main/java/com/facebook/presto/sql/ExpressionUtils.java

```diff
@@ -177,11 +177,6 @@ public static Expression combineConjuncts(Expression... expressions)
     }
 
     public static Expression combineConjuncts(Collection<Expression> expressions)
-    {
-        return combineConjunctsWithDefault(expressions, TRUE_LITERAL);
-    }
-
-    public static Expression combineConjunctsWithDefault(Collection<Expression> expressions, Expression emptyDefault)
```

Comment:
Same for combineDisjuncts?

Reply:
This cannot be done, as they are 2 usages, with different defaults. (i don't understand this, but definitely something purposeful)